### PR TITLE
Adding multi-servo examples

### PR DESCRIPTION
**Pull request by kattni**

### CODE REVIEW

File: examples/servokit_all_servos_sequential.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import time
+from adafruit_servokit import ServoKit
+
+# Set channels to the number of servo channels on your kit.
+# 8 for FeatherWing, 16 for Shield/HAT/Bonnet.
+kit = ServoKit(channels=8)
+
+# Change range to the number of servos connected.
+# They must be on channels within the specified range!
+for i in range(8):
```
Comment (sommersoft) [inline examples/servokit_all_servos_sequential.py]: Could you use `range(len(kit.servo))` here? Then you can drop the warning note...

https://github.com/adafruit/Adafruit_CircuitPython_ServoKit/blob/master/adafruit_servokit.py#L157

File: examples/servokit_all_servos_synchronised.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import time
+from adafruit_servokit import ServoKit
+
+# Set channels to the number of servo channels on your kit.
+# 8 for FeatherWing, 16 for Shield/HAT/Bonnet.
+kit = ServoKit(channels=8)
+
+# Change range to the number of servos connected.
+# They must be on channels within the specified range!
+for i in range(8):
```
Comment (sommersoft) [inline examples/servokit_all_servos_synchronised.py]: Same as above...

File: examples/servokit_all_servos_synchronised.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    kit.servo[i].angle = 180
+time.sleep(1)
+for i in range(8):
```
Comment (sommersoft) [inline examples/servokit_all_servos_synchronised.py]: Once more for good measure? 😄

File: examples/servokit_all_servos_sequential.py (hunk 1 above)
Comment (kattni) [reply examples/servokit_all_servos_sequential.py]: @sommersoft The suggestion works! However, only makes sense if you're working with the exact number of servos as there are channels on the board. So if you connect 4, and say `range(len(kit.servo))` it'll work, but it's sending data to unused channels. 

Maybe it doesn't matter?

Comment (sommersoft) [reply examples/servokit_all_servos_sequential.py]: > Maybe it doesn't matter?

It shouldn't, except to add time to overall loop. 

Like I said, its an option, but guards against any `out of range` occurrences. If you'd rather leave it as is, that's ok too.

**Pull request merged**